JuliaBUGS/README.md

```diff
@@ -46,6 +46,6 @@ For a complete walkthrough, see the [example](https://turinglang.org/JuliaBUGS.j
 ## **Related Tools**
 
 - [**DoodleBUGS**](https://turinglang.org/JuliaBUGS.jl/DoodleBUGS/) — a browser-based interface for drawing BUGS models.
-- [**RJuliaBUGS**](https://mateusmaiads.github.io/rjuliabugs/) — an R interface to JuliaBUGS.
+- [**RJuliaBUGS**](https://mateusmaiads.github.io/rjuliabugs/) — an R interface to JuliaBUGS. This is done using the R package `rjuliabugs`.
```

Comment:
```suggestion
- [**RJuliaBUGS**](https://mateusmaiads.github.io/rjuliabugs/)
```
this should be enough info for now - we can add the cran package after that's through